OWNERS_ALIASES

```diff
@@ -483,13 +483,11 @@ aliases:
     - dchen1107       # Node
     - deads2k         # API Machinery
     - derekwaynecarr  # Node
-    - dghubble        # On Premise
```

Comment:
Not currently an org member
ref: https://cs.k8s.io/?q=dghubble&i=fosho&files=&repos=kubernetes/org